packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/feature_onboarding/services/onboarding_phone_sms_code_service.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:widget_toolkit_otp/widget_toolkit_otp.dart';
+
+import 'user_service.dart';
+
+/// Service used to implement SMS code logic
+class OnboardingPhoneSmsCodeService implements SmsCodeService {
+  OnboardingPhoneSmsCodeService(this._userService);
+
+  final UserService _userService;
+
+  /// Confirm if the entered code is equal to the last send code
+  @override
+  Future<dynamic> confirmPhoneCode(String code) async {
+    final confirmResponse = await _userService.confirmPhoneNumber(code);
```

Comment:
Either implement the service methods or add TODOs of what's expected from the developer

Reply:
Done.